pkg/utils/k8sutil.go

```diff
@@ -1058,3 +1072,14 @@ func addInitContainerAnnotation(dpm *v1beta1.Deployment) error {
 	}
 	return nil
 }
+
+//CascadeDeleteOptions returns option for cascade deletion
+func CascadeDeleteOptions(gracePeriodSeconds int64) *metav1.DeleteOptions {
+	return &metav1.DeleteOptions{
+		PropagationPolicy: func() *metav1.DeletionPropagation {
```

Comment:
Previous comment still applies: remove the func() and just refer to a local variable containing the right (const) value.